src/mame/sunelectronics/markham.cpp

```diff
@@ -2,25 +2,22 @@
 // copyright-holders:Uki
 /*****************************************************************************
 
-    Markham (c) 1983 Sun Electronics
-    Strength & Skill (c) 1984 Sun Electronics
-
     Driver by Uki
 
     TODO:
     - needs merging with ikki.cpp
     - look up schematics for all games
-    - hook up actual SUN 8212 and figure out ROM mode communications
+   	- hook up actual SUN 8212 and figure out ROM mode communications
 
```

Comment:
The content of the change looks fine, but what happened to this line here?  It looks like it’s supposed to be the same, but was a space replaced with a tab or something?  Indents inside comments have indent beyond the level of the opening delimiter replaced with spaces by the srcclean tool for normalisation – if a space was replaced with a tab here, it will get changed back later causing churn.

Reply:
Not sure what happened here, I don't think I reformatted the TODO, but I'll fix it up